src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const source = sourceString;
+  const result = {};
+
+  source
+    .split(';')
+    .map((elem) => elem.split(':'))
+    .map((elem) => elem.map((el) => el.trim()))
+    .filter((elem) => elem.length > 1)
+    .map(([key, value]) => (result[key] = value));
```

Comment:
Using `map` here is not ideal since `map` is intended for transforming arrays and returning a new array. In this case, `forEach` would be more appropriate as you are using it for side effects (modifying the `result` object) rather than transforming the array.